virtcontainers/sandbox.go

```diff
@@ -1228,6 +1228,14 @@ func (s *Sandbox) startVM() error {
 		}
 	}
 
+	// Once the hypervisor is done starting the sandbox,
+	// we want to guarantee that it is manageable.
+	// For that we need to ask the agent to start the
+	// sandbox inside the VM.
+	if err := s.agent.startSandbox(s); err != nil {
```

Comment:
I would put this after the log trace saying `VM started`.

Reply:
Done